### PR TITLE
Add pages

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

File: src/psk-app/psk-app.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 <link rel="import" href="../../../iron-flex-layout/iron-flex-layout.html">
 <link rel="import" href="../../../iron-pages/iron-pages.html">
+<link rel="import" href="../../../iron-pages/iron-pages.html">
```
Comment (robdodson) [inline src/psk-app/psk-app.html]: you're importing this twice

**Comment by robdodson** [comment]
This seems like a good start. We may end up renaming the app from `psk-*` to just like `my-*`, I'm not sure but i think we can go ahead and merge this and deal with renaming stuff later if we decide to do that.

File: src/psk-app/psk-app.html (hunk 1 above)
Comment (abdonrd) [inline src/psk-app/psk-app.html]: Oops! I will remove.

Comment (abdonrd) [inline src/psk-app/psk-app.html]: Fixed here: https://github.com/PolymerElements/polymer-starter-kit/commit/47007581395eeaadcb17f3b08ae75e8be5d242d8

**Pull request merged**